### ISSUE TITLE
Getting System.Management.Automation.RuntimeException

### ISSUE THREAD

**Issue by alxbog** [qetza/vsts-replacetokens-task#21]
Using current 1.4.0 version of extension and latest 2.104.1 agent version on Windows 10 with VSTS.

When the task is run it's failing with the following exception. It looks like it's thrown on each replacement. So we got 6 exceptions that are the same.

```
2016-08-03T17:27:14.5513455Z ##[error]System.Management.Automation.RuntimeException: Unable to find type [Microsoft.TeamFoundation.DistributedTask.Agent.Common.ContextExtensions].
2016-08-03T17:27:14.5513455Z    at System.Management.Automation.TypeOps.ResolveTypeName(ITypeName typeName)
2016-08-03T17:27:14.5513455Z    at System.Management.Automation.Interpreter.FuncCallInstruction`2.Run(InterpretedFrame frame)
2016-08-03T17:27:14.5513455Z    at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
```

**Comment by qetza** [comment]
Hi @alxbog,
Based on first examination, latest version 2.104.1 is based on the new cross platform agent which is not compatible with the task. The task will work with the legacy agent (working on hosted build with version 1.103.1).
I've worked on a new version using typescript and compatible with the new cross platform agent but i cannot release it yet as the new agent doesn't yet allow to access secret variables which where not explicitly linked in an input.
I will try to do some rework on the current version to remove dependency to ContextExtensions.

Thanks for reporting the issue.

**Comment by alxbog** [comment]
@qetza Thanks for your reply.

We just switched to the new cross platform agent. So that's the reason.
As I know the new agent is already a recommended one. So it's definitely must to support it for such a great extension!

Do you know any simple workarounds we can you now without switching back to 1.103.1?

**Comment by qetza** [comment]
Hi @alxbog,
I've published release 1.4.1 which fixes the compatibility issue. It's still using the old powershell host but it's working :smile:

**Comment by alxbog** [comment]
@qetza Thanks! Works now.
